src/photosui.cs

```diff
@@ -580,5 +592,17 @@ interface PHPhotoLibrary_PhotosUISupport
 	{
 		[Export ("presentLimitedLibraryPickerFromViewController:")]
 		void PresentLimitedLibraryPicker (UIViewController controller);
+
+		[NoWatch, NoTV, NoMac, iOS (15,0), MacCatalyst (15,0)]
+		[Export ("presentLimitedLibraryPickerFromViewController:completionHandler:")]
+		void PresentLimitedLibraryPicker (UIViewController controller, Action<NSArray<NSString>> completionHandler);
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PHPickerConfigurationSelection : long
+	{
+		Default = 0,
+		Ordered = 1
```

Comment:
Add a `,` after the last element, because:
* It's valid C# :-)
* It avoid changing this line the next time a member is added (smaller diff to review, better git history)

Reply:
Oops, thanks!